content/en/docs/collector/deployment/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Deployment
+description: Patterns you can apply to run the OpenTelemetry collector
+weight: 2
+---
+
+The OpenTelemetry collector consists of a single binary which you can use in
+different ways, for different use cases. This section describes deployment
+patterns, their use cases along with pros and cons and best practices for
+collector configurations for cross-environment and multi-backend deployments.
+
+## Other information
+
+- GitHub repo [OpenTelemetry Collector Deployment Patterns][gh-patterns]
+- YouTube video [OpenTelemetry Collector Deployment Patterns][y-patterns]
```

Comment:
I would make it clear that this is a talk
```suggestion
- KubeCon NA 2021 Talk ["OpenTelemetry Collector Deployment Patterns"][y-patterns]
```

Reply:
Good suggested change, but drop the quotes.

Reply:
edited, thanks for the feedback :)